.github/workflows/build.yml

```
@@ -24,21 +24,11 @@ jobs:
             fail-fast: false
             matrix:
                 php: [ "7.4", "8.0" ]
-                symfony: [ "^4.4", "5.2.*" ]
-                sylius: [ "~1.9.0", "~1.10.0" ]
+                symfony: [ "^4.4", "5.3.*", "5.4.*" ]
```

Maybe we should also drop support for lower versions of Symfony 5 than 5.4?
```suggestion
                symfony: [ "^4.4", "^5.4" ]
```